app/index.js

```diff
@@ -168,14 +169,16 @@ Generator.prototype.gitIsTheShit = function() {
 			if (err) me.logger.error(err);
 
 			me.logger.verbose('Git init complete');
-			git.add('.', function(err) {
-				if (err) me.logger.error(err);
-			}).commit('Initial Commit', function(err, d) {
-				if (err) me.logger.error(err);
+			done();
+			//git.add('.', function(err) {
+				//if (err) me.logger.error(err);
+
+			//}).commit('Initial WP Commit', function(err, d) {
+				//if (err) me.logger.error(err);
```

Comment:
Well the new fs only writes files at the end of execution. So trying to add and commit files with git breaks when there's no files to add or commit. I tried out doing a lot of small `fs.commit()` calls through each step but ultimately decided it might be more efficient to break those out into bigger chunks. These comments just need to be cleaned up. The initial commit is done after the first `fs.commit()`, after wordpress is installed.

Perhaps it would be better, and closer to the original, to go ahead and `fs.commit()` the gitignore and do an initial commit with just that file.